test/models/admin_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class AdminTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Hey, remember to delete unused code when generating files, controllers, models, etc.